connectors/source.py

```diff
@@ -398,6 +400,11 @@ def hash_id(cls, _id):
 
         return hash_id(_id)
 
+    @classmethod
+    def support_incremental_sync(cls):
+        """Indicates whether the data source supports incremental sync"""
+        return False
```

Comment:
why a classmethod? the `BaseDataSource` class already has two class variables, so it would probably be simpler to have it there too.

class methods are useful if you have to execute some code, here it's just a boolean

Reply:
Makes sense. Will update